components/strava/strava.app.js

```diff
@@ -31,5 +46,56 @@ module.exports = {
         })
       ).data;
     },
+    async createNewActivity($, data) {
+      return (
```

Comment:
I don't believe the extra parens between `return` and `await` are necessary — is that right?

Reply:
Of course. I removed extra parentheses